### PR TITLE
4934362: see also refers to self

### PR DESCRIPTION
**Pull request by prsadhuk**
Some methods and constants has a hyperreference to self in javadoc which is rectified to reference proper methods

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-4934362](https://bugs.openjdk.org/browse/JDK-4934362): see also refers to self

### Reviewers
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**) ⚠️ Review applies to [5c9bd461](https://git.openjdk.org/jdk/pull/11820/files/5c9bd4613fa4081f811cd99cab53ac0b86db9e8a)
 * [Sergey Bylokhov](https://openjdk.org/census#serb) (@mrserb - **Reviewer**)
 * [Alexey Ivanov](https://openjdk.org/census#aivanov) (@aivanov-jdk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11820/head:pull/11820` \
`$ git checkout pull/11820`

Update a local copy of the PR: \
`$ git checkout pull/11820` \
`$ git pull https://git.openjdk.org/jdk pull/11820/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11820`

View PR using the GUI difftool: \
`$ git pr show -t 11820`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11820.diff">https://git.openjdk.org/jdk/pull/11820.diff</a>

</details>

### CODE REVIEW

**Review by prrace — APPROVED** [review]

File: src/java.desktop/share/classes/java/awt/Component.java [hunk 1]
```diff
@@ -2265,7 +2265,7 @@ public void resize(int width, int height) {
      * @param d the dimension specifying the new size
      *          of this component
      * @throws NullPointerException if {@code d} is {@code null}
-     * @see #setSize
+     * @see #getSize
```
Comment (mrserb) [inline src/java.desktop/share/classes/java/awt/Component.java:2268]: The old ref seems to be a shurtcut to the `setSize(int, int)` 
take a look to the current spec
https://docs.oracle.com/en/java/javase/19/docs/api/java.desktop/java/awt/Component.html#setSize(java.awt.Dimension)

File: src/java.desktop/share/classes/java/awt/MediaTracker.java [hunk 2]
```diff
@@ -527,7 +526,7 @@ public boolean checkID(int id) {
      * @return      {@code true} if all images have finished loading,
      *                       have been aborted, or have encountered
      *                       an error; {@code false} otherwise
-     * @see         java.awt.MediaTracker#checkID(int, boolean)
+     * @see         java.awt.MediaTracker#checkID(int, boolean, boolean)
```
Comment (mrserb) [inline src/java.desktop/share/classes/java/awt/MediaTracker.java:529]: int, boolean, boolean is a private method, it should probably refer the checkID(int)

File: src/java.desktop/share/classes/javax/swing/JComboBox.java [hunk 3]
```diff
@@ -155,7 +155,6 @@
      * This protected field is implementation specific. Do not access directly
      * or override. Use the accessor methods instead.
      *
-     * @see #isEditable
```
Comment (mrserb) [inline src/java.desktop/share/classes/javax/swing/JComboBox.java:158]: probably it was planned to point to the isEditable method?

**Review by mrserb — APPROVED** [review]

**Review by aivanov-jdk — APPROVED** [review]

**Pull request closed without merging**